### ISSUE TITLE
Try to work out how this request is reaching Django 

### ISSUE THREAD

**Issue by ddrake** [ddrake/Integrated_Farm_Budget_Tool#157]
http://url1111.ifbt.farm/ls/click?upn=Rn6uuY2iv1LW59HkWR83rs-2FUAPMrXw42Fzv13-2B1epfz0XUod7LQRyej8P-2BvvoI9-2BW37s_mEWvXA1o-2FLV9uv02p4M3WSr3lnvTv987Ro1g-2FVuuHiuMsWWIEilCU6oxAXFsEXevm4qPVVF-2Bl3u-2BYngktCLHfxMj-2BFT9HoZOKe6DBYA2d90ujT8eT4rZz-2FcTEVKYDk99ujBCbLDQG6k93vzc4cOCVIDgh70bZZXxJROJ7iiiTANsX7DBtAnfd0GWDpWDlWrCsgJfoPm3-2FQ-2BxGKkZ-2FWACEA-3D-3D

I would have thought nginx would have filtered it.

**Comment by ddrake** [comment]
I added a block:
```
server {
    server_name _;
    return 444;
}
```
to have Nginx regect any request to a server other than ifbt.farm or www.ifbt.farm.

**Comment by ddrake** [comment]
I think it’s internal — I.e. a request generated by gunicorn.  Need to read the gunicorn documentation.

**Comment by ddrake** [comment]
May have something to do with sendgrid.

**Comment by ddrake** [comment]
I posted to the Django forum about this, but ended up solving my own problem (I think).  I had added a filter to the Nginx settings a while ago to prevent garbage requests with the IP address instead of a domain name from tying up Django.  The problem with that was that requests created during the email confirmation process were throwing an error because they don't have the HTTP_HOST header set.  Normally this header would be set to either 'www.ifbt.farm' or 'ifbt.farm', but instead Nginx was seeing this `/run/gunicorn/sock:` as the host header.  After trying a few other things, I ended up just modifying the filter to allow this "header".  That seems to have solved the issue.  The post is [here](https://forum.djangoproject.com/t/24352).
